includes/utils.php

```diff
@@ -483,6 +483,182 @@ function prepare_meta( $post_id ) {
 	return $prepared_meta;
 }
 
+/**
+ * Generates taxonomy term links for a given post.
+ *
+ * The code is taken from WP_Posts_List_Table::column_default and modified
+ * lightly to work in our context.
+ *
+ * @param string $taxonomy The taxonomy name.
+ * @param object $post     The post object.
+ * @param array  $terms    Optional. Array of terms.
+ * @return string The generated HTML for the taxonomy links.
```

Comment:
Just a blank line before return would be nice.

```suggestion

 * @return string The generated HTML for the taxonomy links.
```

Reply:
Done